build.gradle

```diff
@@ -376,9 +376,9 @@ dependencies {
   integrationTestImplementation sourceSets.main.runtimeClasspath
   integrationTestImplementation sourceSets.test.runtimeClasspath
 
-  testImplementation (group: 'com.github.hmcts', name: 'ccd-test-definitions', version: '7.19.9')
+  testImplementation (group: 'com.github.hmcts', name: 'ccd-test-definitions', version: '7.19.10-prerelease')
```

Comment:
Should be updated with proper release number once hmcts/ccd-test-definitions#61 is released.